index.ts

```diff
@@ -70,3 +70,10 @@ export class Kafka extends BaseKafka {
     return super.consumer({ groupId, ...config })
   }
 }
+
+export {
```

Comment:
Are we allowed to do

```suggestion
export type {
```

to make it clear that we are exporting only TypeScript type declarations, and not actual code?

Also, is it possible to reexport everything except for the client classes that we have customized above?

Reply:
My latest commit should handle both of these points. I tested this locally:

- ran the prepare script
- made a local producer and consumer that import from the generated index.js file
- Started consumer, then trigger producer
- Saw the message successfully delivered.

I can also see all of the kafka js exports in a typescript project now if I am importing from this. 
You can add an import from the locally prepared file, and start typing "KafkaJS" and see all the exports from kafkajs. The whole list should be:

  'AclOperationTypes',
  'AclPermissionTypes',
  'AclResourceTypes',
  'AssignerProtocol',
  'CompressionCodecs',
  'CompressionTypes',
  'ConfigResourceTypes',
  'ConfigSource',
  'Kafka',
  'KafkaJSAggregateError',
  'KafkaJSAlterPartitionReassignmentsError',
  'KafkaJSBrokerNotFound',
  'KafkaJSConnectionClosedError',
  'KafkaJSConnectionError',
  'KafkaJSCreateTopicError',
  'KafkaJSDeleteGroupsError',
  'KafkaJSDeleteTopicRecordsError',
  'KafkaJSError',
  'KafkaJSFetcherRebalanceError',
  'KafkaJSGroupCoordinatorNotFound',
  'KafkaJSInvalidLongError',
  'KafkaJSInvalidVarIntError',
  'KafkaJSInvariantViolation',
  'KafkaJSLockTimeout',
  'KafkaJSMemberIdRequired',
  'KafkaJSMetadataNotLoaded',
  'KafkaJSNoBrokerAvailableError',
  'KafkaJSNonRetriableError',
  'KafkaJSNotImplemented',
  'KafkaJSNumberOfRetriesExceeded',
  'KafkaJSOffsetOutOfRange',
  'KafkaJSPartialMessageError',
  'KafkaJSProtocolError',
  'KafkaJSRequestTimeoutError',
  'KafkaJSSASLAuthenticationError',
  'KafkaJSServerDoesNotSupportApiKey',
  'KafkaJSStaleTopicMetadataAssignment',
  'KafkaJSTimeout',
  'KafkaJSTopicMetadataNotLoaded',
  'KafkaJSUnsupportedMagicByteInMessageSet',
  'PartitionAssigners',
  'Partitioners',
  'ResourcePatternTypes',
  'logLevel'